<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Withdraw</class>
 <widget class="QWidget" name="Withdraw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="le_AccountNumber">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_AccountPassword">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_WithdrawAmount">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_AccountNumber">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>32</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>계좌 번호:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_AccountPassword">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>62</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>비밀 번호:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_WithdrawAmount">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>92</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>출금액:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Withdraw">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>출금</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ExitWithdraw">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
